lib/Backend/BackendUtils.cpp

```diff
@@ -327,6 +327,9 @@ runtime::RuntimeBundle::create(const IRFunction &F,
                                MemoryAllocator &constantAllocator,
                                MemoryAllocator &placeholderAllocator,
                                MemoryAllocator &activationsAllocator) {
+
+  bool contiguous = (&constantAllocator == &placeholderAllocator &&
```

Comment:
```suggestion
  // If all allocators refer to the same underlying allocator, Constants, Placeholders and activations will be allocated contiguously. The maximum memory usage reported by the allocator for each kind of storage will include the memory usage of all previously allocated types of storage and needs to be adjusted accordingly.
  bool contiguous = (&constantAllocator == &placeholderAllocator &&
```

Or something like that.